data_structures/Trie/Trie.java

```diff
@@ -0,0 +1,67 @@
+import java.util.* ;
+
+class trie_node
```

Comment:
```suggestion
class TrieNode
```
Java uses PascalCase